test/e2e/lib/hooks/browser-logs.js

```diff
@@ -2,19 +2,19 @@
  * External dependencies
  */
 import fs from 'fs/promises';
+import { logging } from 'selenium-webdriver';
 
 /**
  * Internal dependencies
  */
-import { getBrowserLogs, getPerformanceLogs } from '../driver-helper';
 import { generatePath } from '../test-utils';
 
 export const saveBrowserLogs = async () => {
 	try {
 		await Promise.allSettled(
 			[
-				[ () => getBrowserLogs( this.driver ), 'console.log' ],
-				[ () => getPerformanceLogs( this.driver ), 'performance.log' ],
+				[ () => this.driver.manage().logs().get( logging.Type.BROWSER ) ],
+				[ () => this.driver.manage().logs().get( logging.Type.PERFORMANCE ) ],
```

Comment:
The second element of the array, the log destination, is missing
```suggestion
				[ () => this.driver.manage().logs().get( logging.Type.BROWSER ), 'console.log' ],
				[ () => this.driver.manage().logs().get( logging.Type.PERFORMANCE ), 'performance.log' ],
```

Reply:
Oops! 😬 Nice catch 👍